go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/coreos/go-semver
+
+go 1.8
```

Comment:
q: This go version seems a little old. Shouldn't it be `go 1.5/1.16`?

Reply:
The codebase hasn't changed since 2017 and a number of other projects depend on it.  I didn't want to bump the minimum supported Go version solely by adding a `go.mod`.

We don't CI Go versions that old, though.  GitHub Actions can test Go back to 1.9.  We could add tests for those versions, but I don't think we _actually_ want to support them... anyone have thoughts?  @lucab?